k8s/yaml/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ts-bridge
+spec:
+  selector:
+    matchLabels:
+      app: ts-bridge
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: ts-bridge
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+      containers:
+        - image: REPLACE_ME
+          imagePullPolicy: Always
```

Comment:
This isn't ideal. The image tag should be unique so shouldn't need re-pulling. If you use `:latest` as the tag, k8s will always pull anyway. Removing this line would be saner IMHO.

Reply:
That was a leftover from debugging, removed.